src/main/java/Duke.java

```diff
@@ -1,10 +1,98 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String line;
+        Scanner in = new Scanner(System.in);
+
+        System.out.println("____________________________________________________________");
+        System.out.println(" Hello! I'm Duke");
+        System.out.println(" What can I do for you?");
+        System.out.println("____________________________________________________________");
+
+        Task[] list = new Task[100];
+        int nextEmptyIndex = 0;
+
+        boolean done = false;
+
+        while (!done) {
```

Comment:
Perhaps you could separate your functionality into multiple submethods to make the code more readable and modular?